lizmap/server/lizmap_filter.py

```diff
@@ -47,43 +47,67 @@ def __init__(self, server_iface: 'QgsServerInterface') -> None:
 
     def requestReady(self):
         try:
+            # Get QGIS Project path
             config_path = self.iface.configFilePath()
             if not os.path.exists(config_path):
+                # QGIS Project path does not exist as a file
+                # The request can be  evaluated by QGIS Server
                 return
 
+            # Get Lizmap config path
             config_path += '.cfg'
             if not os.path.exists(config_path):
+                # Lizmap config path does not exist
+                # The request can be  evaluated by QGIS Server
                 return
 
+            # Get Lizmap config
             cfg = None
             with open(config_path, 'r') as cfg_file:
                 try:
                     cfg = json.loads(cfg_file.read())
                 except Exception:
```

Comment:
You must log an error or at least a warning, otherwise an error in the config file will go unoticed. In all cases, it is a bad practice to let a exception silent unless you have have a really good reason.

Reply:
Wrong line, should be line 60.

Reply:
log massage added